### PR TITLE
Turn off default httpavailabilitystrategy for source-google-analytics-v4 (GA)

### PR DESCRIPTION
**Pull request by erohmensing**
See https://github.com/airbytehq/airbyte/pull/21888/ for description

### CODE REVIEW

**Review by evantahler — APPROVED** [review]
Your change is 👍, but I don't really know how to interpret the SAT failure:

```
SKIPPED [1] ../usr/local/lib/python3.9/site-packages/source_acceptance_test/tests/test_core.py:94: The previous and actual specifications are identical.
SKIPPED [1] ../usr/local/lib/python3.9/site-packages/source_acceptance_test/tests/test_core.py:377: The previous and actual discovered catalogs are identical.
```

**Comment by erohmensing** [comment]
SAT is successful - those are for the skipped checks. You'll see it on other successes and failures (e.g. [here](https://github.com/airbytehq/airbyte/pull/22001))

The new QA checks are failing because it can't tell if the [changelog has been updated](https://github.com/airbytehq/airbyte/actions/runs/4028383627/jobs/6925220831#step:15:52). Pretty sure this is due to the [weird naming on the google analytics sources vs their documentation](https://airbytehq-team.slack.com/archives/C02UF50V9HA/p1674167266006479). This should be fixed (change the docs names?) but I'm not sure all the ramifications of where those docs files are used and would probably do that separately

File: docs/integrations/sources/google-analytics-universal-analytics.md [hunk 1]
```diff
@@ -159,6 +159,7 @@ Incremental sync is supported only if you add `ga:date` dimension to your custom
 
 | Version | Date       | Pull Request                                             | Subject                                                                                      |
 |:--------|:-----------|:---------------------------------------------------------|:---------------------------------------------------------------------------------------------|
+| 0.1.34  | 2022-01-27 | [22006](https://github.com/airbytehq/airbyte/pull/22006) | Set `AvailabilityStrategy` for streams explicitly to `None`                                                     |
```
Comment (erohmensing) [inline docs/integrations/sources/google-analytics-universal-analytics.md:162]:
```suggestion
| 0.1.34  | 2023-01-27 | [22006](https://github.com/airbytehq/airbyte/pull/22006) | Set `AvailabilityStrategy` for streams explicitly to `None`                                                     |
```
Unless it caught this, which I don't think it did, but good to catch anyway!

**Pull request merged**